lib/model/model.py

```diff
@@ -64,7 +68,10 @@ def get_response(self, message: schemas.Message) -> schemas.GenericItem:
                 result = self.process(message)
                 Cache.set_cached_result(message.body.content_hash, result)
             except Exception as e:
-                return self.handle_fingerprinting_error(e)
+                if isinstance(e, PrestoBaseException):
```

Comment:
since we are (I think) nicely bubbling up errors to this level,  should logger.exception() here so that it will still get logged to Sentry from the presto side?

Reply:
I think this exists on line 58, no?

Reply:
oh, I didn't realize that is what capture_custom_messages does.  In Presto we are explicitly capturing message so it will go to Sentry only and not normal python logs?  If so maybe that capture message argument should be 'error' instead of 'info'

Reply:
done! (changed info to error)

overall, I also think it's better to log errors when they happen. makes debugging easier, and we also won't lose the line number on which the error logging happens. for all of classycat errors, I made sure we log the errors where they happen.